UndertaleModTool/Scripts/Community Scripts/UndertaleWithJSONs.csx

```diff
@@ -58,44 +55,27 @@ void IncProgressLocal()
         IncrementProgress();
 }
 
-string GetFolder(string path) {
-    return Path.GetDirectoryName(path) + Path.DirectorySeparatorChar;
-}
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////
 
 ImportGMLString("gml_Script_scr_change_language", @"");
 ImportGMLString("gml_Script_scr_84_load_map_json", @"");
-if (Data.GeneralInfo.Major < 2) // Undertale PC (GMS1)
-{
-    ImportGMLString("gml_Script_textdata_en", @"
-    if (variable_global_exists(""text_data_en""))
-        ds_map_destroy(global.text_data_en);
-    global.text_data_en = scr_84_load_map_json(program_directory + ""\lang\"" + ""lang_en.json"");");
-    ImportGMLString("gml_Script_textdata_ja", @"
-    if (variable_global_exists(""text_data_ja""))
-        ds_map_destroy(global.text_data_ja);
-    global.text_data_ja = scr_84_load_map_json(program_directory + ""\lang\"" + ""lang_ja.json"");");
-}
-else
-{
-    ImportGMLString("gml_Script_textdata_en", @"
-    if (variable_global_exists(""text_data_en""))
-        ds_map_destroy(global.text_data_en);
-    global.text_data_en = scr_84_load_map_json(program_directory + ""\lang\\"" + ""lang_en.json"");");
-    ImportGMLString("gml_Script_textdata_ja", @"
-    if (variable_global_exists(""text_data_ja""))
-        ds_map_destroy(global.text_data_ja);
-    global.text_data_ja = scr_84_load_map_json(program_directory + ""\lang\\"" + ""lang_ja.json"");");
-}
```

Comment:
Have you checked if `""\lang\\""` can be `""\lang\""` for GMS 2?

Reply:
this shouldnt matter. 
it essentially wants to do `program directory + \lang\ + lang_ja.json` (or lang_en). Hence why I replaced it with a slash, as that then also works for all operating systems properly instead of just Windows. and these two paths `program_directory/lang` or `program_directory/////lang` are the same.